js/localization/default_date_names.js

```diff
@@ -1,19 +1,16 @@
-var iteratorUtils = require("../core/utils/iterator");
+import iteratorUtils from "../core/utils/iterator";
 
-var MONTHS = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
-    DAYS = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"],
-    PERIODS = ["AM", "PM"],
-    QUARTERS = ["Q1", "Q2", "Q3", "Q4"];
+const MONTHS = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"], DAYS = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"], PERIODS = ["AM", "PM"], QUARTERS = ["Q1", "Q2", "Q3", "Q4"];
```

Comment:
IMO previous formatting was better

Reply:
It was not perfect lebab config. I'll fix.